ig/input/fsh/nbs-obr-segment.fsh

```diff
@@ -2,28 +2,31 @@ Logical: OBRSegment
 Id: segment-obr-logical-model
 Title: "OBR Segment"
 Description: "The OBR Segment"
-* setOBRId 1..1 SU string "OBR-1 Set ID"
-* OBRplacerOrderNumber 1..1 SU string "OBR-2 Placer Order Number"
-* fillerOrderNumber 1..1 SU string "OBR-3 Filler Order Number"
+* setOBRId 1..1 SU positiveInt "OBR-1 Set ID"
+* placerOrderNumber 1..1 SU string "OBR-2 Placer Order Number"
+* placerOrderNumberEntityIdentifier 1..1 SU string "OBR-2.1 Placer Order Number Entity Identifier"
+* placerOrderNumberNamespaceID 1..1 SU string "OBR-2.2 Placer Order Number Namespace ID"
+* placerOrderNumberUniversalID 1..1 SU string "OBR-2.3 Placer Order Number Universal ID"
+* placerOrderNumberUniversalIDType 1..1 SU code "OBR-2.4 Placer Order Number Universal ID Type"
 * universalServiceIdentifier 1..1 SU string "OBR-4 Universal Service Identifier"
-* observationDateTime 1..1 SU string "OBR-7 Observation Date/Time"
-* observationEndDateTime 1..1 SU string "OBR-8 Observation End Time"
-* specimenID 1..1 SU string "OBR-15 or OBR-16  Specimen Id"
-* specimenType 1..1 SU string "OBR-15 or OBR-16 Specimen Type"
-* specimenCollectionDateTime 1..1 SU string "OBR-7 Specimen Collection Date/Time"
+* universalServiceIdentifierIdentifier 1..1 SU string "OBR-4.1 Universal Service Identifier Identifier"
+* universalServiceIdentifierText 1..1 SU string "OBR-4.2 Universal Service Identifier Text"
+* universalServiceIdentifierNameOfCodingSystem 1..1 SU string "OBR-4.3 Universal Service Identifier Name Of Coding System"
+* universalServiceIdentifierOriginalText 1..1 SU string "OBR-4.9 Universal Service Identifier Original Text"
+* observationDateTime 1..1 SU dateTime "OBR-7 Observation Date/Time"
 // other optional fields
 
 
 // Provenance Section begins
 Instance: segment-obr-logical-model-history-create
```

Comment:
Please keep the original Provenance instance definition and create a new one for the new update.  Each Provenance instance becomes a line on the history tab for the OBR segment model.